phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/DragableRepositoryListCell.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * 
+ */
+package org.phoenicis.javafx.views.mainwindow.settings;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListCell;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.input.Dragboard;
+import javafx.scene.input.TransferMode;
+import javafx.scene.layout.GridPane;
+
+/**
+ * @author marc
+ *
+ */
+public class DragableRepositoryListCell extends ListCell<String> implements ChangeListener<Number> {
+
+	/**
+	 * What should be done after this cell has been dragged
```

Comment:
This comment is not needed, however, the corresponding javadoc can be written on the constructor